web-marketplace/src/legacy-pages/Dashboard/Dashboard.EditProfile.tsx

```diff
@@ -144,21 +278,49 @@ export const EditProfile = () => {
     }
   }, [activeAccount, reactQueryClient, wallet?.address]);
 
+  const refreshOrganizationData = useCallback(
+    async (daoAddr?: string) => {
+      if (!daoAddr) return;
+      await Promise.all([
+        reactQueryClient.invalidateQueries({
+          queryKey: ['dao-config', daoAddr],
+        }),
+        reactQueryClient.invalidateQueries({
+          queryKey: ['organization-profile', daoAddr],
+        }),
```

Comment:
those query keys does not actually refer to any existing query so this does not do anything